### PR TITLE
CodePeer Training

### PR DESCRIPTION
**Pull request by leogermond**
- codepeer: fix linter going overboard
- codepeer: add style presentation slide
- codepeer: small intro refacto
- fix CWE attribute
- codepeer tutorial lab
- codepeer radar lab
- codepeer: cruise lab
- codepeer: intro remove ref to baseline, add review
- codepeer: Condition Predet. reword, fix example
- codepeer: Loop Doesnt ... Normaly: reword
- codepeer: Unused Assignment: reword
- codepeer: typo in subp name
- codepeer: fix annotation list order
- codepeer: quiz
- build codepeer lab and slides
- github: only build codepeer [REMOVE ME]
- codepeer: radar: disable server by default
- codepeer: fix quiz function example

### CODE REVIEW

**Review by frank-at-adacore — DISMISSED** [review]
I did not review the lab source files - I'm assuming after 3 classes they should be pretty good :)

File: courses/codepeer/010_codepeer.rst [hunk 1]
```diff
@@ -1288,10 +1396,12 @@ Condition Predetermined
...
 
-+ Indicates loops that either
++ The loop will never complete its **exit condition**
++ Causes can be
 
-  + runs forever
-  + fails to terminate normally
+  + Exit condition is always :ada:`False`
+  + An exception is raised
+  + The exit condition code is dead
```
[the remark this answers is not in the document]
Comment (leogermond) [reply courses/codepeer/010_codepeer.rst:1404]: "code should be dead code"? That sounds super weird

[the remark this answers is not in the document]
Comment (frank-at-adacore) [reply courses/codepeer/010_codepeer.rst:1404]: I missed the first "code"
How about "The exit condition is in dead code"?

File: courses/codepeer/010_codepeer.rst [hunk 2]
```diff
@@ -787,6 +804,67 @@ Precondition
 
 | ``high: precondition (conditional check) failure on call to precondition.call: requires X < 0``
 
+------
+Quiz
+------
+
+* Which check will be raised with the following?
```
[the remark this answers is not in the document]
Comment (leogermond) [reply courses/codepeer/010_codepeer.rst:811]: Done

File: courses/codepeer/010_codepeer.rst [hunk 3]
```diff
@@ -1288,10 +1396,12 @@ Condition Predetermined
 Loop Does Not Complete Normally
 ---------------------------------
 
-+ Indicates loops that either
++ The loop will never complete its **exit condition**
```
[the remark this answers is not in the document]
Comment (leogermond) [reply courses/codepeer/010_codepeer.rst:1399]: Done

File: courses/codepeer/010_codepeer.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (leogermond) [reply courses/codepeer/010_codepeer.rst]: I reworded that to unreachable, LMK if that fits

**Review by frank-at-adacore — APPROVED** [review]

**Pull request merged**